apps/klasses/models.py

```diff
@@ -116,4 +117,7 @@ def homeworks_completely_graded(self):
                         student_graded = True
                     if not student_graded:
                         return False
-                return True
+                hw_graded = True
+            if not hw_graded:
+                return False
```

Comment:
can just do return `hw_graded` here because if it's false we'll be returning false anyway.

Reply:
Yeah we should not be returning in that case.